src/frontend/src/pages/ProjectDetailPage/ProjectEditContainer/ProjectEditContainer.tsx

```diff
@@ -45,7 +45,7 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ proj, exitE
   const allUsers = useAllUsers();
   const { mutateAsync } = useEditSingleProject(proj.wbsNum);
 
-  const [crId, setCrId] = useState(query.get('crId') || -1);
+  const [crId, setCrId] = useState(query.get('crId') || NaN);
```

Comment:
not a fan of `NaN`, you can just do empty string then you'll have to change one line below